common/privatekey.go

```diff
@@ -47,14 +46,14 @@ func (pk PrivateKey) Sign(hash []byte) Sign {
 	if err == nil {
 		sign = *BytesToSign(sig)
 	} else {
-		panic(fmt.Sprintf("Sign Failed, reason : %v.\n", err.Error()))
+		err = DefaultLogger.Errorf("Sign Failed, reason : %v.\n", err.Error())
```

Comment:
should use fmt.Errorf. DefaultLogger maybe unavailable in test case

Reply:
All test cases run success

Reply:
done